maintenance/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM nginx
```

Comment:
Can we use the alpine one?

Reply:
Tried that. It results in an image that's about 95MB vs 197MB ... so that's not bad, but then the `filebeat` module doesn't include the `nginx` module, and I think `logrotate` isn't fully configured. I ended up deciding it wasn't worth the effort.